pyop2/compilation.py

```diff
@@ -50,6 +50,29 @@
 from pyop2.exceptions import CompilationError
 
 
+class UnknownVersion(Version):
+    """ packaging.version.Version won't let you create an "unknown"
+    version, so this class compares False to other instances of version
+    """
+    def __init__(self):
+        pass
+
+    def __repr__(self):
+        return f"<Version('{self}')>"
+
+    def __str__(self):
+        return "unknown"
+
+    def __gt__(self, other):
+        return False
+
+    def __eq__(self, other):
+        return False
+
+    def __lt__(self, other):
+        return False
```

Comment:
This seems like a massive hack. Does it not make more sense to define a `is_unknown_version` variable and then have
```py
if is_unknown_version:
    return
```
at the top of the necessary functions?

Reply:
Perhaps, I was just trying to recreate the behavior of `version.LooseVersion("unknown")`